src/cli/ImportFeedCommand.ts

```diff
@@ -91,8 +91,8 @@ export class ImportFeedCommand implements CLICommand {
   /**
    * Create the last_file table (if it doesn't already exist)
    */
-  private createLastProcessedSchema(): Promise<void> {
-    return this.db.query(`
+  private async createLastProcessedSchema(): Promise<void> {
+    await this.db.query(`
```

Comment:
this change does nothing

Reply:
It is required because a Promise<any[], any> can't be assigned into Promise<void> in Typescript. It explicitly states that the query result is to be discarded.

Reply:
I see, did you work around it?

Reply:
Errr, sorry for the confusion I hadn't considered the types. Maybe put it back to how you had it

Reply:
I have just pushed a commit to revert this change.